### DISCUSSION TITLE
Online keyboard. I can't get it to work!

### DISCUSSION THREAD

**Discussion by juankstylez** [nutgram/nutgram#302] [General]
You can help ?

I'm trying to follow the documentation with this keyboard and it doesn't work for me, I don't get the error !

what am i missing?

https://prnt.sc/klLtoN4z26PV  Telegram.php

https://prnt.sc/6PPv8h6c8KLK  telegram 

https://prnt.sc/U1PNi9SWs4Pq Console error :(

**Comment by Lukasss93** [comment]
ops! you need to use this method: onCallbackQueryData

**Reply by juankstylez** [reply]
No Work :(  https://prnt.sc/kw2Dph2PXkKz

**Comment by juankstylez** [comment]
Something is wrong ! It says I have to pass it an array but I tried and no response :(

No Work :( https://prnt.sc/kw2Dph2PXkKz

After that error, I try to pass it as an array and it just doesn't respond, it doesn't do anything ! What can be ?

**Reply by Lukasss93** [reply]
ops, sorry again 😅

```php
$bot->answerCallbackQuery([
    'text' => 'my answer'.
    'show_alert' => true,
]);
```

**Reply by juankstylez** [reply]
Thanks
